src/lib/hooks/useConnect/setupConnection.ts

```diff
@@ -86,17 +91,30 @@ export async function setUpConnection({
         logger?.info?.('SendbirdProvider | useConnect/setupConnection/configureSession', sessionHandler);
         newSdk.setSessionHandler(sessionHandler);
       }
-      // to check if code is released version from rollup and *not from storybook*
-      // see rollup config file
-      // @ts-ignore
-      if (IS_ROLLUP === IS_ROLLUP_REPLACE) {
-        logger?.info?.('SendbirdProvider | useConnect/setupConnection/setVersion', APP_VERSION_STRING);
-        newSdk.addExtension('sb_uikit', APP_VERSION_STRING);
-      }
+
+      logger?.info?.('SendbirdProvider | useConnect/setupConnection/setVersion', { version: app_version });
+      newSdk.addExtension('sb_uikit', app_version);
+
       const connectCbSucess = (user: User) => {
         logger?.info?.('SendbirdProvider | useConnect/setupConnection/connectCbSucess', user);
         sdkDispatcher({ type: INIT_SDK, payload: newSdk });
         userDispatcher({ type: INIT_USER, payload: user });
+
+        newSdk.getUIKitConfiguration()
+          .then(config => {
+            const payload = snakeToCamel(config.json)?.uikitConfigurations;
```

Comment:
Should we handle situation where `config` is undefined? Or is it safe from SDK side?

Reply:
Right if the config is undefined then it'd be an error case which should be caught by catch block.